### DISCUSSION TITLE
滑手機開暗黑模式的好與壞

### DISCUSSION THREAD

**Discussion by yuchien-lin** [1101-GS4719-C/Python-Community-GS4719-C#1881] [General]
開暗黑模式對我們有好有壞，好處是對光線敏感的人觀看時較為舒適、較省電，然而壞處是人眼的機制較習慣在亮光下觀察暗色，而非反過來在暗黑的背景裡閱讀亮色，所以這會導致有些人在閱讀時感到不適，再來是長期使用暗黑模式，眼睛的肌群其實會更容易疲勞，所以其實暗黑模式並全非如同我們所想，他也可能也會在你沒發現的情況下，帶給你壞處。
資料來源:[https://3c.ltn.com.tw/news/38491](url)

**Comment by RavenJiaSin** [comment]
在寫程式時，我個人還是習慣使用深色背景，淺色的IDE介面真的好刺眼。這可以說是一種習慣，或是說其實是一種傳統(有人會跳出來說:淺色是邪教吧)，另外，如果是傳統LCD的螢幕，黑色還是會需要點亮背光元件，所以不見得會比較省電。

**Comment by TobywithDino** [comment]
我個人很喜歡深色模式，以前有在IDE上面試著調整好看的樣式，就有調過背景顏色，那時候的深色模式其實並沒有全黑，所以就天才的想：要是我把顏色調成全黑不就更好看了嗎，後來試過才發現那樣子超難看，眼睛也沒有比較舒服

**Comment by sorn32** [comment]
我覺得深色模式真的很好看(￣∇￣)
而且偶然發現手機可以根據現在實際的時間，調整淺色或深色的部分，我覺得這樣也不錯，不用一直都維持同一種模式

**Comment by chuanmin0317** [comment]
我現在的手機會在特定的時間幫我調整淺色模式或是深色模式，有些應用程式像是youtube和chrome也有這個功能，我覺得這個功能的確讓我在長時間使用手機時不會那麼的疲憊

**Comment by Yanxxx7979** [comment]
我最開始使用深色模式是用在lineㄟ 在賴的聊天室使用上 我個人是用眼會最不舒服的
因為需要盯著文字又要打字 鍵盤的字一直跳動 使用深色的主題會讓我的眼睛更沒有負擔

**Comment by YANRONG0123** [comment]
我還是比較喜歡使用深色模式，無論其帶給我的任何傷害，原因是亮色模式帶給我的感覺過於刺眼，以至於我難以辨識其中的文字，看久了眼睛會非常痠痛，需要好好休息才能彌補。

**Comment by willychiuchiu** [comment]
白天我覺得一般模式會比較方便閱讀，但天黑時黑暗模式看起來確實比較舒適一些，尤其是在夜晚或光線較暗的地方，如果打開是白光眼睛真的會被刺到，有時候還會被嚇到，很不蘇湖，所以對我來說比較理想的方式應該是隨當下的環境做切換

**Comment by YuanWang08** [comment]
如果要長時間的看著螢幕，我會選擇黑暗模式，眼睛感覺比較舒服。但若只是短時間就能完成的任務，我會把黑暗模式關掉，明亮的顏色會讓我比較有動力且較能夠專注。

**Comment by thomasxshen** [comment]
我比較喜歡深色模式，因為比較不傷眼睛，現代人的文明病就是一直滑手機，深色模式減少眼睛所接觸的藍光，但有時也會看不太見。

**Comment by yxin214** [comment]
原來暗黑模式有這麼多好處，一直以為只有顏色的區別而已！我沒有使用暗黑模式單純覺得淺色模式的版面比較好看，但身邊多數人都偏好深色模式。但我想不論使用哪種模式，適度用眼才是維護眼睛的根本之道。

**Comment by wanka2030** [comment]
我個人還是prefer淺色模式，因為淺底配黑字才是我習慣的，雖然打程式的時候反而會比較常用深色底，因為可以配合酷酷的特效配件然後不同的顏色以及物件才能夠清楚，但心中還是喜歡淺色模式啦!

**Comment by happy246810** [comment]
在光線充足的條件下，我傾向淺色的螢幕，因為這樣眼睛比較舒服，而相反的，在光線不足的時候，我偏向使用深色的螢幕，因為深色螢幕比淺色螢幕較不刺激眼睛，但我還是建議在陰暗的環境不要用3C，因為對眼睛不好。

**Comment by jylin34** [comment]
我認為好與壞主要還是跟手機的螢幕類型有點關係，如果是lcd的螢幕，調深色模式不會比較省電，螢幕調暗會對眼睛比較好。如果是oled，深色模式會比較省電，調暗螢幕反而會更傷眼睛，調暗會讓螢幕閃爍頻率增加，盯久了眼睛就會感覺到酸痛，所以最傷眼睛的方式就是晚上睡前滑手機(

**Comment by LinChloe** [comment]
個人覺得淺色模式較不容易讓我想睡覺，可能是因為太刺眼的關係。
以前一直是淺色派，但最近3C用多後，眼睛疲勞一直提升，現在只好用深色模式了。

**Comment by William08290829** [comment]
我自己是看當下的使用環境去做更改，如果環境燈光很亮我會用原本的模式，但燈光暗的時候真的是不用暗黑模式不行，眼睛真的會瞎掉。雖可能會容易導致疲勞但真的沒辦法。

**Comment by tshsu04224** [comment]
我個人是習慣了深色模式，所有能調的一切，都要是深色的那種，現在真的沒辦法適應白色模式了，就覺得這樣看字的時候背景好刺眼，有種站在大太陽底下盯著天空看很久的感覺，白色模式真的用不久就感覺眼睛疲勞，想休息了；但相反的深色模式就會不知不覺讓我一直盯著手機或電腦看，久久沒有休息，我想，是否能有自覺適度的讓眼睛放鬆，這也是一個我必須反思的問題。

**Comment by YOGA012** [comment]
這是我第一次聽說暗黑模式有這些潛在的壞處，真是讓我大開眼界。之前一直覺得暗黑模式比較舒適，也能省電，但沒想到它會讓眼睛更容易疲勞。這樣看來，以後可能需要在不同情況下切換使用模式，來更好地保護眼睛。謝謝你分享這些資訊，真是受益匪淺！

**Comment by woohsin** [comment]
暗黑模式有其優缺點。好處包括對光線敏感者更舒適、較省電，特別是在OLED螢幕上。然而，壞處也不容忽視：人眼更習慣在亮光下觀察暗色，而在暗色背景下閱讀亮色文字可能導致不適。此外，長期使用暗黑模式可能增加眼睛疲勞。因此，暗黑模式並非如我們所想的那樣完美，可能在不知不覺中對眼睛造成負擔。使用者應根據自身情況選擇適合的顯示模式，並定期休息眼睛，以避免不必要的眼睛疲勞。

**Comment by Hahahahbla** [comment]
你對暗黑模式的分析非常全面。確實，暗黑模式有其優點和缺點，這取決於使用者的需求和使用情境。總之，暗黑模式有其優點，但也需要根據實際情況和個人需求來使用。希望這些建議能幫助你更好地利用暗黑模式，達到最佳的視覺舒適度。

**Comment by OG-Matcha** [comment]
我自己比較常使用深色模式，寫程式寫久了不管甚麼應用都是黑的才順眼，突然看到白底對眼睛是爆擊傷害，另外在進到比較暗的環境時可以不用切換，直接使用，但相對來說也會讓我使用 3C 產品的時間拉長。

**Comment by NcuJerryKao** [comment]
這篇文章探討了暗黑模式的優缺點，以下是幾個建設性建議，幫助提升文章的結構、內容和清晰度：
1. 引言部分的強化：
- 引言可以更具吸引力和信息性，簡要介紹暗黑模式的背景和普及情況，並指出文章的主要討論點。
- 例如：「隨著科技的發展，暗黑模式（Dark Mode）已成為許多應用程式和操作系統中的熱門選擇。然而，暗黑模式對我們的視覺健康到底有什麼影響呢？」
2. 段落結構：
- 將文章分成更易於理解的段落，每個段落集中討論一個具體的觀點。例如：
1. 引言：介紹暗黑模式的背景和普及情況。
2. 好處：詳細說明暗黑模式對光線敏感者和省電的好處。
3. 壞處：討論暗黑模式對眼睛疲勞和閱讀不適的影響。
4. 結論和建議：總結文章並提出具體的建議。
3. 過渡和連接詞：
- 使用過渡句和連接詞來連接不同段落，使文章流暢。例如，「接下來，我們來看看暗黑模式的好處。」或「然而，暗黑模式也有一些壞處需要注意。」
4. 詳細說明和分析：
- 在描述暗黑模式的好處和壞處時，可以更詳細地解釋其背後的原因和機制。例如，詳細說明暗黑模式如何減少光線刺激和省電，以及為什麼會導致眼睛疲勞和閱讀不適。
- 例如：「暗黑模式可以減少光線對眼睛的刺激，特別適合對光線敏感的人。此外，使用暗黑模式還可以節省電量，延長設備的使用時間。」
5. 語言簡化和澄清：
- 某些段落中的語言較為簡單，可以考慮稍微豐富和澄清。例如，「人眼的機制較習慣在亮光下觀察暗色，而非反過來在暗黑的背景裡閱讀亮色」可以改為「人眼的視覺機制更適應在明亮的背景下觀察暗色文字，而在暗黑的背景下閱讀亮色文字可能會導致視覺不適。」
6. 引用和支持：
- 若能引用一些具體的研究或專家觀點來支持論點，將大大增強文章的權威性。例如，可以引用視覺健康研究或專家觀點來支持「暗黑模式對眼睛疲勞」的討論。
7. 個人經驗和反思：
- 若有相關的個人經驗或反思，可以分享在文章中，這樣可以讓讀者更容易產生共鳴。例如，描述自己使用暗黑模式的經歷和感受。
8. 結論和建議強化：
- 在結論部分，再次強調文章的主要觀點，並總結文章的重要結論。這有助於讀者記住文章的核心思想。
- 例如：「總結來說，暗黑模式有其優點，例如減少光線刺激和節省電量，但也可能導致視覺不適和眼睛疲勞。使用暗黑模式時，我們應該根據自身需求和感受來選擇，並注意適時調整以保護視力。」
9. 讀者互動：
- 在文章結尾，可以提出一些問題或邀請讀者分享他們的觀點和經驗，以增加互動性和參與感。
- 例如，「你有使用暗黑模式的經驗嗎？你覺得暗黑模式對你有什麼影響？歡迎在評論區分享你的意見。」
通過這些建議，文章將變得更加結構化、清晰且具說服力，讀者也會更容易理解和應用其中的信息。

**Comment by titan92601** [comment]
我認為暗黑模式有利有弊。它舒適節能，但對部分人來說閱讀不適且可能導致眼睛疲勞。因人而異，使用者應留意模式對自身的影響，並適時調整使用方式。

**Comment by alexender0104** [comment]
在開暗黑模式我發現可以減少眼睛疲勞，在低光環境中，暗黑模式可以減少屏幕光線的刺眼程度，從而減少眼睛疲勞，特別是暗黑模式可以減少屏幕的反光和眩光，尤其是在光線較弱的環境中。

**Comment by Shang621** [comment]
開啟暗黑模式有著雙重影響。好處在於減少眼睛疲勞，特別是在夜晚或光線暗的環境中。這可以幫助保護眼睛，提升使用手機的舒適度。此外，暗黑模式還能節省電量，延長手機使用時間，對於追求效率和節能的人來說是一大優勢。然而，暗黑模式也有其不利之處。長時間使用暗黑模式可能導致對亮度較低的環境適應能力降低，使得在明亮光線下的閱讀變得困難。此外，對於一些人來說，暗黑模式可能不夠舒適，他們更喜歡亮白的界面。開啟暗黑模式有其利有其弊，每個人可以根據自己的偏好和需求來選擇是否使用。

**Comment by 110101501** [comment]
我自己喜歡使用深色模式。不然過去使用淺色模式時，只要一打開手機畫面都會先被畫面的亮光亮到。
自從調整到深色模式後，就比較沒有這方面的問題，對眼睛的刺激沒有這麼大。

**Comment by yubo0oo** [comment]
暗黑模式確實有其優缺點。對於光線敏感的人來說，它提供了更舒適的閱讀體驗並節省電量。然而，對於某些人來說，在暗色背景下閱讀亮色文字可能不太舒適，且長時間使用可能導致眼睛疲勞。因此，使用暗黑模式時應該注意眼睛的舒適度，並適時調整。

**Comment by jeff110303549** [comment]
開暗黑模式對我們有好有壞，好處是對光線敏感的人觀看時較為舒適、較省電，然而壞處是人眼的機制較習慣在亮光下觀察暗色，而非反過來在暗黑的背景裡閱讀亮色，所以這會導致有些人在閱讀時感到不適，再來是長期使用暗黑模式，眼睛的肌群其實會更容易疲勞，所以其實暗黑模式並全非如同我們所想，他也可能也會在你沒發現的情況下，帶給你壞處。

**Comment by sandyho0307** [comment]
我覺得暗黑模式對於那些在夜間使用電子設備的人來說，確實是一個福音。當周圍環境較暗時，暗黑模式可以減少屏幕的光線刺激，讓眼睛感到更加舒適。

**Comment by 710388** [comment]
感覺很不錯阿，我平常都用黑暗模式，眼睛感覺比較舒服

**Comment by cassy0201** [comment]
我自己也是習慣用深色模式,這樣在晚上滑手機的時候可以欺騙自己沒那麼傷眼睛哈哈哈

**Comment by 155Julia** [comment]
我比較喜歡暗色模式，因為晚上閱讀時比較不刺眼。晚上使用暗色模式的確讓眼睛感覺比較舒服，但在長時間使用後，我也發現眼睛會變得疲勞。雖然暗黑模式在某些情況下很有用，但我認為還是需要根據不同的環境和需求靈活切換，這樣才能最大程度地保護眼睛。

**Comment by hbh10601** [comment]
蛤!竟然是這樣，我原本以為深色看起來比較舒服對眼睛也比較好，結果和我所想的不一樣，但是Coding的時候用亮色真的好刺眼，從深色模式出來之後我幾乎所有App、軟體都用深色，尤其Eclipse那個白色真的很亮。

**Comment by YeeDaniel** [comment]
自從被室友大推深色模式後，直接回不去了，推爛深色模式，不管是瀏覽器還是IDE，白色模式個人認為很傷眼睛，太亮了。

**Comment by yubo0oo** [comment]
黑暗中瞳孔放大，藍光直射眼底
以前的人常常是在光線不足的情況下看書，背景光線不足，會讓睫狀肌調節視物更吃力、近視增加、慢慢影響視力。呂大文說，黑暗中使用手機卻大不相同，書本本身不會發光，但是手機本身是個發光體，它是一種強光，而且是很傷眼的藍光，黑暗中使用手機時，人的瞳孔放到最大，藍光毫無阻隔直接進到視網膜會造成更大的危害。

**Comment by poiu6fi** [comment]
我不管是瀏覽器 LINE 或者是vscode都是用深色模式，覺得比較好看也不容易刺眼，尤其是在晚上關燈的時候，不過缺點是LINE有些貼圖的字會看不到

**Comment by OHoln** [comment]
個人已經用習慣深色模式了，每次用別人電腦是淺色模式的時候都像被閃光彈閃到。而且也有使用螢幕掛燈，所以不會有光源不足的問題

**Comment by yjpxi** [comment]
我是都用暗黑模式，特別是在低光環境中使用時更舒適，也可以節省手機電量。但是，暗黑模式並不是對所有人都適用，特別是長時間需要進行細節閱讀或工作的情況下，可能要用淺色模式。

**Comment by MingWei174** [comment]
我習慣用暗黑模式進行要長時間觀看螢幕的動作，例如:觀看影片或做報告等，因為亮色模式看螢幕太久我會雙眼刺痛很不舒服，甚至在眼睛離開螢幕的瞬間還會有眩光的是視覺暫留效果!
但事實上常常使用暗黑模式眼睛的肌群其實會更容易疲勞，所以還是依照個人的習慣及感覺調整螢幕亮度即可!

**Comment by Nana1218** [comment]
我自已是喜歡用暗黑模式，因為不只可以節省手機電力，我在看螢幕的時候也比較不會那麼刺眼，尤其是在晚上較暗的地方使用時，暗黑模式對我來說是很必要的。

**Comment by kingsophy** [comment]
我個人是比較喜歡淺色模式，因為用久了淺色模式，如果切成深色模式的話眼睛會不太舒服，身體也不習慣，因此就一直用淺色模式直到現在。

**Comment by hunteryang0108** [comment]
雖說黑暗模式設計的初衷就是讓人在黑暗環境下使用手機比較不那麼刺眼，不過在黑暗中使用手機本來就是一個不健康的行為，我們應該盡量減少這種情況的發生，不然很可能讓自己的眼睛產生如青光眼之類的病變。

**Comment by CtyceT** [comment]
無論是手機、電腦介面還是Vscode、Github、Discord等程式我都會使用黑暗模式。但我個人使用黑暗模式主因並非為了眼睛的健康考慮，反而是為了手機和電腦電池的健康。使用黑暗模式後明顯比淺色模式省電，而若真的要護眼的話我會開啟護眼模式讓螢幕泛黃再使用。

**Comment by Jerryguo888** [comment]
我個人比較喜歡開黑暗模式，因為像是在使用google或者在寫程式時，我個人覺得使用深色模式看起來會比較舒服。隨然我比較喜歡深色模式，但我們應該根據自身的視覺需求來選擇合適的顯示模式。適時調整，並定期休息眼睛，才能更好地保護視力。

**Comment by irenesunnn** [comment]
視網膜剝離患者路過
以前都開深色模式，手機亮度調暗(感覺光少就對眼睛沒這麼刺激
現在才知道手機亮度過低會讓眼睛更用力地去對焦在內容上，容易造成眼睛疲勞
還是要依照環境光去調整螢幕亮度和模式更能保護眼睛哦
另外推薦iphone用戶可以設置螢幕使用距離，讓自己不要太低頭滑手機
